### DISCUSSION TITLE
Interoception VSD/Grid

### DISCUSSION THREAD

**Discussion by sagostine** [UNC-Project-Open-AAC/OS-DPI#213] [Show and tell]
Hi All! So excited that I finally got the Interoception VSD/Grid to where I wanted it to be! 

[screen-capture (4).webm](https://github.com/UNC-Project-Open-AAC/OS-DPI/assets/123391336/a71a3bf5-414e-4147-8b56-564e9e20b7f3)

[FINAL Introception_Grid_7_28_23 - Copy.zip](https://github.com/UNC-Project-Open-AAC/OS-DPI/files/12197130/FINAL.Introception_Grid_7_28_23.-.Copy.zip)

**Comment by aetkie** [comment]
Susie, this looks great! Would you mind adding a few sentences to this post re: how did you come up with this design, who the intended user would be, what purpose does this board serve, what is introception? (for those of us non-OS people 😆)

**Comment by sagostine** [comment]
Absolutely!

I came up with this design after one of our meetings when we were talking about the possibilities of the OSDPI. I thought about the higher functioning generally autistic kids I worked with in the clinic and how much they struggled to identify what was going on in their body. Interoception is the feelings inside your body, the easiest being when you need to go to the bathroom, when you are hungry, but can also include things like how anxiety or anger or fatigue presents. Being able to identify these feelings, connect them to emotions and finally connect them to a self-regulation strategy is hard work and very conceptual. By making this VSD/grid, I made these conceptual steps visual. I also gave a limited number options (which can be changed/modified for the user) which I had found to be helpful as a starting off point for describing these feelings/emotions/strategies. The intended user is anyone who needs assistance identifying the feelings in their body, connecting them to emotions and self-regulation strategies but I definitely had my previous clients in mind. Although they were generally verbal, I do feel this board would still be very helpful for them and could be used across populations :)

**Comment by aetkie** [comment]
Thanks for adding this context @sagostine. So the interoception UI design is intended to be used as in multiple steps?
1. identify the body part
2. identify the associated feeling/emotion
3. can then follow up with an action "i want a high five"

I agree, this UI could be useful for investigating a whole range of research questions with a variety of individuals.

**Reply by sagostine** [reply]
Correct! 

There is the option to just use one part if needed but it was made with the intention of the person going through each step!
